core/indigo-core/molecule/src/base_molecule.cpp

```diff
@@ -278,6 +278,12 @@ void BaseMolecule::_mergeWithSubmolecule_Sub(BaseMolecule& mol, const Array<int>
         reaction_atom_exact_change[mapping[i]] = mol.reaction_atom_exact_change[i];
     }
 
+    if (skip_flags & COPY_BOND_DIRECTIONS)
+    {
+        _bond_directions.expandFill( edgeEnd() + mol.edgeEnd(), 0);
+    } else
+      _bond_directions.expandFill(  mol.edgeEnd(), 0);
+
```

Comment:
It's definitely required to run clang-format on changed files.

Reply:
fixed